scripts/output/comparison/compareScenarios2.R

```diff
@@ -4,7 +4,20 @@
 # |  AGPL-3.0, you are granted additional permissions described in the
 # |  REMIND License Exception, version 1.0 (see LICENSE file).
 # |  Contact: remind@pik-potsdam.de
-# ---- Define set of runs that will be compared ----
+
+
+
+
+# Header ------------------------------------------------------------------
+
+
+# Make sure the script is called via output.R.
+if (!exists("outputdirs")) stop("outputdirs does not extist. Please call comapreScenarios.R via output.R, which defines outputdirs.")
```

Comment:
define what variables are expected to exist here. Maybe allow to load them from command line?